test/scripts/dynamic-fee-teal-test.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+date '+dynamic-fee-teal-test start %Y%m%d_%H%M%S'
+
+set -e
+set -x
+export GOPATH=$(go env GOPATH)
+
+TEMPDIR=$(mktemp -d)
+trap "rm -rf $TEMPDIR" 0
+
+NETDIR=${TEMPDIR}/net
+
+if [ ! -z $BINDIR ]; then
+    export PATH=${BINDIR}:${PATH}
+fi
+
+goal network create -r ${NETDIR} -n tbd -t ${GOPATH}/src/github.com/algorand/go-algorand/test/testdata/nettemplates/TwoNodes50EachFuture.json
+
+goal network start -r ${NETDIR}
+
+# replaces prior trap0
+trap "goal network stop -r ${NETDIR}; rm -rf ${TEMPDIR}" 0
+
+export ALGORAND_DATA=${NETDIR}/Node
+
+ACCOUNT=$(goal account list|awk '{ print $3 }')
```

Comment:
you're using pipe here, which is ok, but you need to ensure that you propogate the error code from the goal and not overriding it with the exit code from awk. use "set -o pipefail" to do that.

Reply:
done